api/client.gen.go

```diff
@@ -245,6 +245,13 @@ func (c *APIClient) callAPI(request *http.Request) (*http.Response, error) {
 		}
 		log.Printf("\n%s\n", string(dump))
 	}
+	if resp.Header.Get("trace-sampled") == "true" {
```

Comment:
This file is code-gen'd from a template, so you're going to need to edit the template and then generate this file.

It should be pretty easy to add this to https://github.com/influxdata/influx-cli/blob/4c0fae3a4c0d77df0fcbd91583df8b246d488e30/api/templates/client.mustache#L173

Then  you can run `make openapi` to re-generate this client file.

Reply:
```console
$ ./etc/generate-openapi.sh
Error opening file "/api/contract/openapi/src/oss/paths/health.yml"
ENOENT: no such file or directory, open '/api/contract/openapi/src/oss/paths/health.yml'
```

Reply:
ah gitsubmodules...

Reply:
```console
$ cat .gitmodules
[submodule "openapi"]
        path = api/contract/openapi
        url = ../openapi
```

Reply:
https://github.com/influxdata/influx-cli/blob/main/api/contract/README.md#updating-the-api-contract

Reply:
yeah I didn't expect it to be a relative path to another checkout. I'll send another PR

Reply:
UH, TIL git knows how to handle relative paths on remotes too now? wow

Reply:
anyway sorry for the noise; now the change is in the moustache template and I issued `make openapi`